backend/src/main/java/com/crm/dto/response/CarResponse.java

```diff
@@ -0,0 +1,21 @@
+package com.crm.dto.response;
+
+import com.crm.model.db.CarTypeEntity;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class CarResponse {
+
+    private Long id;
+
+    private String vin;
+    private String registrationNumber;
+    private String brand;
+    private String model;
+    private Integer productionYear;
+    private Integer mileage;
+    private String description;
+    private CarTypeEntity carTypeEntity;
```

Comment:
Czy tutaj potrzebna jest cała encja? Może można tylko zwrócić stringa name. Jeżeli jednak to pole jest wynikiem działania klasy  ModelMapper, może można napisać swój Getter który zwraca Stringa?

Reply:
I was wondering about it too. Initially it was a String, but then I changed to entity because of ModelMapper. I wanted to see your comments on this firstly.

Reply:
I would recommend remove that empty String. But, maybe someone more experienced share opinion too.